src/serac/physics/utilities/functional/integral_utilities.hpp

```diff
@@ -230,6 +232,29 @@ struct qf_result<lambda_type, x_t, u_du_t, qpt_data_type, std::enable_if_t<!std:
                                     std::add_lvalue_reference_t<qpt_data_type>>;
 };
 
+/**
+ * @brief derivatives_ptr access
+ *
+ * Templating this will allow us to change the stride-access patterns more consistently
+ *
```

Comment:
maybe rename `quadrature_coalescing` to `row_major` or provide a `@tparam` description for people who aren't familiar with GPU memory access terminology.